GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSwitch.java

```diff
@@ -32,13 +31,13 @@
  */
 public class GVRSwitch extends GVRBehavior
 {
-    static private long TYPE_SWITCH = (System.currentTimeMillis() & 0xfffffff);
+    static private long TYPE_SWITCH = 0;
```

Comment:
If we need a quick fix I guess you could shift GVRSwitch.class.hashCode() to the left and & it with currentTimeMillis.